test/acceptance/tests.py

```diff
@@ -8,13 +8,13 @@
 from functools import wraps
 from pyinstrument import Profiler
 
-from nose.plugins.attrib import attr
-from bok_choy.web_app_test import WebAppTest
-from bok_choy.promise import BrokenPromise, EmptyPromise
-from auto_auth import AutoAuthPage
-from pages import (
+from acceptance.auto_auth import AutoAuthPage
+from acceptance.pages import (
     SubmissionPage, AssessmentPage, GradePage, StaffAreaPage
 )
+from bok_choy.web_app_test import WebAppTest
+from bok_choy.promise import BrokenPromise, EmptyPromise
+from nose.plugins.attrib import attr
```

Comment:
Nit: Can you put a `from __future__ import absolute_import` at the top here to verify that implicit relative imports (like `from auto_auth import AutoAuthPage`) don't come back?